### DISCUSSION TITLE
Which version of flowbite-react is the current documentation for?

### DISCUSSION THREAD

**Discussion by kevinorin** [themesberg/flowbite-react#620] [Q&A] [answered]
I understand flowbite-react and the documentation is a work in progress, but I was having issues getting Tabs to work according to the current documentation.

Using flowbite-react 0.3.8 with tabs in this format did not work:
`<Tabs.Group>
      <Tabs.Item active={true} title="Overview">
`

Following my intellisense I got it working using:
` <TabsComponent>
            <TabItem active={true} title="Overview" >`

I also can't get properties to work such as  style="underline"

Which version do I need to use in order for the documented examples to work as documented? https://flowbite-react.com/tabs

**Comment by kevinorin** [comment, marked as answer]
Resolved, I was using Tabs for my component name as well.

**Comment by tulup-conner** [comment]
For posterity, the documentation at flowbite-react.com reflects whatever the latest commit is on the main branch.

With the current CI, that means sometimes there can be documentation that is newer than what is available through npm. We intend to fix that permanently after the next release.
